src/cmd/root.go

```diff
@@ -48,7 +48,7 @@ var rootCmd = &cobra.Command{
 	},
 	Short: lang.RootCmdShort,
 	Long:  lang.RootCmdLong,
-	Args:  cobra.MaximumNArgs(1),
+	Args:  cobra.ExactArgs(1),
```

Comment:
```suggestion
	Args:  cobra.MaximumNArgs(1),
```

`zarf` is a thing - it prints the logo and help